packages/edit-widgets/src/components/sidebar/widget-areas.js

```diff
@@ -59,7 +59,7 @@ export default function WidgetAreas( { selectedWidgetAreaId } ) {
 						<Button
 							href={ addQueryArgs( 'customize.php', {
 								'autofocus[panel]': 'widgets',
-								return: 'themes.php?page=gutenberg-widgets',
+								return: 'widgets.php',
```

Comment:
I think this should this be a ternary based on whether the version of WordPress core includes widgets or not?

- `! function_exists( 'wp_use_widgets_block_editor' )` should return to `themes.php?page=gutenberg-widgets`
- `function_exists( 'wp_use_widgets_block_editor' )` should return to `widgets.php`

Would be good to add comments as well saying we can remove the URL `themes.php?page=gutenberg-widgets` in WordPress v5.9.

Reply:
Oh duh! This is JavaScript. Well, I guess we could either add it as an editor setting. Or themes.php?page=gutenberg-widgets could have a redirect if the right page is widgets.php (which might not be a bad idea anyway.)

Reply:
> Well, I guess we could either add it as an editor setting.

Sucks that we'd have to maintain this editor setting forever and ever though given that once Gutenberg requires 5.8 the string will be `widegts.php` and never change.

> Or themes.php?page=gutenberg-widgets could have a redirect if the right page is widgets.php (which might not be a bad idea anyway.)

Hm, but then we'd be adding plugin-specific cruft to Core. I'm not a big fan :) 

---

To be honest I think this button is so small and out of the way that it's not a big deal if it doesn't work properly in the plugin for a few releases.

But, I just had an idea: we can use `GUTENBERG_PHASE` to tell if we're in the plugin!

Reply:
Wait lol why I don't I just use `location.pathname`. Done in 4c57f3ad12.